### PR TITLE
cleanups, clarifications, fixups for alignRequests

### PR DESCRIPTION
**Pull request by Dieterbe**

### CODE REVIEW

File: metric_tank/query_engine.go [hunk 1]
```diff
@@ -57,16 +57,17 @@ func alignRequests(reqs []Req, aggSettings []aggSetting) ([]Req, error) {
 	}
 	tsRange := (reqs[0].to - reqs[0].from)
 
+	// note: not all series necessarily have the same raw settings, will be fixed later
```
Comment (woodsaj) [inline metric_tank/query_engine.go]: not all series having the same raw settings is already taken into consideration. That is why we use `minInterval`

Comment (Dieterbe) [inline metric_tank/query_engine.go]: i know, and this comment clarifies that that will be addressed later on, to improve clarity to anyone reading the code from top to bottom.

Comment (woodsaj) [inline metric_tank/query_engine.go]: are you saying that "fixed later" means "further down" and not "some time in the furture"

Comment (Dieterbe) [inline metric_tank/query_engine.go]: right, i'll update that

**Comment by woodsaj** [comment]
@Dieterbe this PR doesnt build

**Comment by Dieterbe** [comment]
@woodsaj oops, just rebased and repushed. should be good now.

**Pull request merged**